src/cdk/drag-drop/drop.ts

```diff
@@ -81,27 +74,31 @@ export class CdkDrop<T = any> implements OnInit, OnDestroy {
   @Input() id: string = `cdk-drop-${_uniqueIdCounter++}`;
 
   /** Locks the position of the draggable elements inside the container along the specified axis. */
-  @Input() lockAxis: 'x' | 'y';
+  @Input('cdkDropLockAxis') lockAxis: 'x' | 'y';
 
   /**
    * Function that is used to determine whether an item
    * is allowed to be moved into a drop container.
    */
-  @Input() enterPredicate: (drag?: CdkDrag, drop?: CdkDrop) => boolean = () => true;
+  @Input('cdkDropEnterPredication')
```

Comment:
Why is it renamed from predicate to predication?

Reply:
Good question, it's a typo. I'll sort it out.